lib/crypto/Crypto.js

```diff
@@ -72,7 +72,19 @@ module.exports = class Crypto {
 		const power = new BigNumber(key.d());
 		const mod = new BigNumber(key.n());
 
-		return (modPow(base, power, mod)).toBEBuffer().toString('base64');
+		//Somehow sometimes we have a 514 byte hex key that starts with "00". In that case use only the last 512 bytes
+		var hexString = modPow(base, power, mod).toBEBuffer().toString('hex');
```

Comment:
`modPow(base, power, mod).toBEBuffer()` could we cache this buffer in case no trimming is needed.

Reply:
See last commit. That should be a bit cleaner and do the same in the end.